docs/UserGuide.md

```diff
@@ -67,6 +74,40 @@ $ add lesson CS2113 Lecture -d MON -s 1pm -e 3pm
     
 ```
 
+### Listing Tasks or Lessons - `list`
+
+`list` can be used to perform any of the following actions:
+- list all tasks and lessons
+- list all tasks only
+- list all lessons only
```

Comment:
> Use * for bullet-points (not -).
>
> RATIONALE: Although both work, * closer to the final outcome.

[Source](https://se-education.org/guides/conventions/markdown.html)

Minor standard violation here. I also notice the same issues below.

Reply:
We can ignore this for now.